engine/src/main/scala/cromwell/engine/workflow/WorkflowManagerActor.scala

```diff
@@ -102,9 +103,13 @@ class WorkflowManagerActor(config: Config)
 
   private val donePromise = Promise[Unit]()
 
+  val isRestartable = config.getConfig("system").getBooleanOr("workflow-restart", default = true)
+  val serverMode = CromwellServer.isServerMode
```

Comment:
This will actually instantiate that CromwellServer object. What that comment means is that it won't actually exist until the first thing reaches in and touches it.